src/legacy/core_plugins/metrics/public/components/index_pattern.js

```diff
@@ -62,11 +63,16 @@ export const IndexPattern = props => {
               id="tsvb.indexPatternLabel"
               defaultMessage="Index pattern"
             />)}
+            helpText={(model.default_index_pattern && !model[indexPatternName] && <FormattedMessage
+              id="tsvb.indexPattern.searchByDefaultIndex"
+              defaultMessage="Default index pattern is used. To query all indexes use *"
```

Comment:
I think it should use `indices` as plural of `index` - but I wasn't sure and searched for it in the Kibana codebase - `2800` vs `141` in favor of `indices`, so I think we should go with it :)